src/question.js

```diff
@@ -20,12 +20,21 @@ router.post('/check/:qno(\\d+)?', middleware.isAuthenticated, (req, res) => {
   var { qno } = req.params;
   const { answer } = req.body;
   const { lastQuestionAllowed } = req.user;
+  const {score} = req.user;
   if (!qno) qno = lastQuestionAllowed;
   if (qno > lastQuestionAllowed) {res.sendStatus(403); return false;}
   models.Question.findOne({where : { qno }})
     .then(question => {
       if (question) {
-        if (question.answer == answer && qno ==lastQuestionAllowed) req.user.update({lastQuestionAllowed: lastQuestionAllowed + 1});
+        if (question.answer == answer && qno ==lastQuestionAllowed)
+        {
+	  req.user.update({ score: score + 10});
+          req.user.update({ lastQuestionAllowed: lastQuestionAllowed + 1 });
```

Comment:
Instead od having multiple updates, just have one update query, as in `update({ x : 2, y: 5})`